src/tests.rs

```diff
@@ -17,6 +17,14 @@ use rand_core::OsRng;
 use std::collections::HashSet;
 use proptest::prelude::*;
 
+// Configure proptest to run fewer cases for faster testing
+// Default is 256 cases, we'll use 8 for quicker feedback  
```

Comment:
nit: remove trailing whitespace

```suggestion
// Default is 256 cases, we'll use 8 for quicker feedback
```